app/javascript/Pages/components/CardPopular.scss

```diff
@@ -0,0 +1,49 @@
+$primary-color: #6600cc;
+$dark-color: #252636;
+$white-color: #ffffff;
+$gray-color: #aaaaaa;
+
+.Media {
```

Comment:
**issue (blocking)**: consistency 🙈 css classes should not have uppercase leters